src/Geta.NotFoundHandler.Admin/Areas/GetaNotFoundHandlerAdmin/Pages/Base/AbstractSortablePageModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Geta.NotFoundHandler.Admin.Areas.GetaNotFoundHandlerAdmin.Pages.Models;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Geta.NotFoundHandler.Admin.Areas.GetaNotFoundHandlerAdmin.Pages.Base;
+
+public abstract class AbstractSortablePageModel : PageModel
+{
+    public string SortColumn { get; set; }
```

Comment:
IMHO this should be nullable?

Reply:
If nullable reference types are disabled, string behaves like it traditionally does in C# and can hold null.